src/main/java/uk/ac/ebi/subs/filecontentvalidator/service/FileType.java

```diff
@@ -0,0 +1,26 @@
+package uk.ac.ebi.subs.filecontentvalidator.service;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public enum FileType {
+    FASTQ, BAM, CRAM, VCF, FLATFILE;
```

Comment:
Can we leave FLATFILE out for now, it's not on our roadmap yet?

Reply:
Yes. I removed it

Reply:
👍